lrlex/Cargo.toml

```diff
@@ -27,6 +27,7 @@ vergen = { version = "8", default-features = false, features = ["build"] }
 [dependencies]
 cfgrammar = { path = "../cfgrammar", version = "0.13" }
 lrpar = { path = "../lrpar", version = "0.13" }
+proc-macro2 = "1.0"
```

Comment:
This should probably be in ../Cargo.toml and use `proc-macro2.workspace = true` here.

Reply:
Fixed in [acf3134](https://github.com/softdevteam/grmtools/pull/494/commits/acf3134e53ab0675d5c7a093340b32b52df2af3f)